src/attributes/views/AttributeCreate/AttributeCreate.tsx

```diff
@@ -54,6 +57,39 @@ function areValuesEqual(
   return a.name === b.name;
 }
 
+function getSimpleAttributeData(
+  data: AttributePageFormData,
+  values: AttributeValueEditDialogFormData[]
+) {
+  return {
+    ...data,
+    metadata: undefined,
+    privateMetadata: undefined,
+    storefrontSearchPosition: parseInt(data.storefrontSearchPosition, 0),
+    values: values.map(value => ({
+      name: value.name
+    }))
+  };
+}
+
+function getFileAttributeData(
+  data: AttributePageFormData,
+  values: AttributeValueEditDialogFormData[]
+) {
+  return {
+    ...data,
```

Comment:
can we use getSimpleAttributeData here and spread it so the properties are not duplicated